apps/crn-server/serverless.ts

```diff
@@ -1189,6 +1192,13 @@ const serverlessConfig: AWS = {
         IS_CONTENTFUL_ENABLED: 'true',
       },
     },
+    sendSlackAlert: {
+      handler: './src/handlers/send-slack-alert.handler',
+      condition: 'IsProd',
```

Comment:
Why would we run the lambda in dev or branch environment to do nothing, when we can not create it to begin with?

Reply:
because then we can actually test our infrastructure elsewhere. IMHO its dangerous to have such conditions in place, there is no reason why the lambda shouldn't run in dev or PR.